3542/0/ruby_koans/about_scoring_project.rb

```diff
@@ -0,0 +1,84 @@
+# rubocop:disable Lint/MissingCopEnableDirective, Metrics/AbcSize
+# rubocop:disable Lint/UnneededCopDisableDirective
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Greed is a dice game where you roll up to five dice to accumulate
+# points.  The following "score" function will be used to calculate the
+# score of a single roll of the dice.
+#
+# A greed roll is scored as follows:
+#
+# * A set of three ones is 1000 points
+#
+# * A set of three numbers (other than ones) is worth 100 times the
+#   number. (e.g. three fives is 500 points).
+#
+# * A one (that is not part of a set of three) is worth 100 points.
+#
+# * A five (that is not part of a set of three) is worth 50 points.
+#
+# * Everything else is worth 0 points.
+#
+#
+# Examples:
+#
+# score([1,1,1,5,1]) => 1150 points
+# score([2,3,4,6,2]) => 0 points
+# score([3,4,5,3,3]) => 350 points
+# score([1,5,1,2,4]) => 250 points
+#
+# More scoring examples are given in the tests below:
+#
+# Your goal is to write the score method.
+
+def score(dice)
+  # You need to write this method
+  score = 0
+  score += dice.count(1) >= 3 ? (1000 + (dice.count(1) - 3) * 100) : dice.count(1) * 100
+  score += dice.count(5) >= 3 ? (500 + (dice.count(5) - 3) * 50) : dice.count(5) * 50
+  [2, 3, 4, 6].each { |x| dice.count(x) >= 3 ? score += x * 100 : 0 }
```

Comment:
The rules of this game says there're 5 rolls only but anyway
In case when there're 6 twos you'll get `200` instead of `400` and so on.
Please consider to refactor this method following this links
https://ruby-doc.org/core-2.6.3/Enumerable.html#method-i-reduce
https://apidock.com/ruby/Object/tap